content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Get Pentest Updates with Webhooks"
+linkTitle: "Webhooks"
+weight: 10
+description: >
+  Set up webhook notifications for your organization.
+---
+
+{{% pageinfo %}}
+Configure webhooks to subscribe to real-time updates for your pentests.
+{{% /pageinfo %}}
+
+With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
```

Comment:
I feel like this is the only time we need to mention API (possible exception, whatever CX-I comes up with)

From a UI User PoV, I'm not sure they care that our Webhooks are based on our API.

Reply:
@mjang-cobalt I think our integrations are aimed at advanced users (in many cases, developers). So I think it's OK to keep this info,

Reply:
Some additional info:

The webhooks use a skinny payload, containing only the public id of the entity. This is for security reasons. 

So to be able to retrieve information that actually was updated about the entity, the users will need to use e.g. our public API. 

Maybe we should add some documentation on how to retrieve information about a webhook event with the API?
cc: @eugenerevzin

Reply:
@inikolov89 I think adding more info about APIs is out of scope for this task since we're only describing the Webhooks UI in this guide.
I plan to add API use cases to our Product Documentation with more details about using webhook APIs. You can see an example use case for assets here: https://developer.cobalt.io/apiusecases/create_asset/